### ISSUE TITLE
Failed to invoke a chain twice based on XataChatMessageHistory in the same session.

### ISSUE THREAD

**Issue by IvoRacious** [langchain-ai/langchainjs#5636]
### Checked other resources

- [X] I added a very descriptive title to this issue.
- [X] I searched the LangChain.js documentation with the integrated search.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in LangChain.js rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).

### Example Code

```Ruby
/// <reference types="https://esm.sh/v135/@supabase/functions-js@2.4.1/src/edge-runtime.d.ts" />
import { BufferMemory } from "https://esm.sh/langchain@0.2.4/memory";
import { ChatOpenAI } from "https://esm.sh/@langchain/openai@0.1.0";
import { ConversationChain } from "https://esm.sh/langchain@0.2.4/chains";
import { XataChatMessageHistory } from "https://esm.sh/@langchain/community@0.2.5/stores/message/xata";
import { BaseClient } from "https://esm.sh/@xata.io/client@0.28.4";

const getXataClient = () => {
    const xata = new BaseClient({
      databaseURL: xata_db_url,
      apiKey: xata_apiKey,
      branch: xata_branch,
    });
    return xata;
  };
  
const memory = new BufferMemory({
  chatHistory: new XataChatMessageHistory({
    table: xata_table_name,
    sessionId: new Date().toISOString(), // Or some other unique identifier for the conversation
    client: getXataClient(),
    createTable: false, // Explicitly set to false if the table is already created
  }),
});
  
const model = new ChatOpenAI({apiKey: openai_apiKey, model: openai_model, temperature: 0});
const chain = new ConversationChain({ llm: model, memory });
  
const res1 = await chain.invoke({ input: "Hi, I'm Jim" });
console.log({ res1 });
const res2 = await chain.invoke({ input: "Who am I?"});
console.log({ res2 });
```

### Error Message and Stack Trace (if applicable)

```bash
[Info] {
  res1: {
    response: "Hello Jim! It's nice to meet you. How can I assist you today?"
  }
}
[Error] SyntaxError: "[object Object]" is not valid JSON
    at JSON.parse (<anonymous>)
    at https://esm.sh/v135/@langchain/community@0.2.5/esnext/stores/message/xata.js:2:2067
    at Array.map (<anonymous>)
    at r.map (https://esm.sh/v135/@xata.io/client@0.29.5/esnext/client.mjs:4:46285)
    at o.getMessages (https://esm.sh/v135/@langchain/community@0.2.5/esnext/stores/message/xata.js:2:2011)
    at eventLoopTick (ext:core/01_core.js:64:7)
    at async x.loadMemoryVariables (https://esm.sh/v135/langchain@0.2.4/esnext/memory.js:2:2174)
    at async T._formatValues (https://esm.sh/v135/langchain@0.2.4/esnext/chains.js:2:2329)
    at async T.invoke (https://esm.sh/v135/langchain@0.2.4/esnext/chains.js:2:821)
    at async Object.handler (file:///home/deno/functions/chain/index.ts:56:16)
```

### Description

I just duplicated the code in the official document of Xata Chat Memory:
https://js.langchain.com/v0.1/docs/integrations/chat_memory/xata/ and tried to run it locally. 

I can successfully invoke the chain when creating a new session, and can find records of human and AI messages in my xata database. 

However, when I tried to invoke it in the same session, I failed and got ``` [Error] SyntaxError: "[object Object]" is not valid JSON ``` .

It seems like a Javascript object was wrongly passed to the ```JSON.parse```function. However, the code is too complicated to understand. Please tell me how to debug and solve it.

### System Info

 ```Deno1.44.0```

**Comment by IvoRacious** [comment]
I've checked in xata database that all of the ```additionalKwargs`` are empty JSON objects. The problem must exist in the process of   retrieving and parsing. How could I verify whether the data being passed is correctly formatted?

**Comment by IvoRacious** [comment]
I think I found the problem.
The previous xata table I linked to was originally created by xata pakages of python version.
So when I set ```createTable``` to ```true``` to create a new table using xata packages of js version, it works.
The true reason involve too many codes in too many packages, which is too complicated, and I don't want to investigate further. Howerver, I would be appreciated if someone can tell me. Thank you!
